app/src/components/notifications/PageNotifications.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <TmPage data-title="My alerts" hide-header>
+  <TmPage data-title="My alerts">
     <TmDataLoading v-if="$apollo.loading && notifications.length === 0" />
```

Comment:
use TmPage loading?

Reply:
yep. this page was not implemented correctly 😞